specification/ai/data-plane/HealthInsights/stable/2024-04-01/openapi.json

```diff
@@ -0,0 +1,3056 @@
+        "description": "Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus2.api.cognitive.microsoft.com).",
+        "required": true,
+        "type": "string",
+        "format": "uri",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "ApiKeyAuth": []
+    }
+  ],
+  "securityDefinitions": {
+    "ApiKeyAuth": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "tags": [
+    {
+      "name": "RadiologyInsights"
+    }
+  ],
+  "paths": {
+    "/radiology-insights/jobs/{id}": {
+      "get": {
+        "operationId": "RadiologyInsights_GetJob",
+        "tags": [
+          "RadiologyInsights"
+        ],
+        "summary": "Get Radiology Insights job details",
+        "description": "Gets the status and details of the Radiology Insights job.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The unique ID of the job.",
+            "required": true,
+            "type": "string",
+            "minLength": 3,
+            "maxLength": 36,
+            "pattern": "^[a-zA-Z0-9][a-zA-Z0-9-_]+[a-zA-Z0-9]$"
+          },
+          {
+            "name": "expand",
+            "in": "query",
+            "description": "A comma separated list of related properties to be included in line with the job.",
+            "required": false,
+            "type": "string",
+            "collectionFormat": "csv",
+            "enum": [
+              "jobData"
+            ],
+            "x-ms-enum": {
+              "name": "ExpandQueryParamsType",
+              "modelAsString": true,
+              "values": [
+                {
+                  "name": "JobData",
+                  "value": "jobData",
+                  "description": "Use this value to expand the job with jobData property."
+                }
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/RadiologyInsightsJob"
+            },
+            "headers": {
+              "Retry-After": {
+                "type": "integer",
+                "format": "int32",
+                "description": "The Retry-After header can indicate how long the client should wait before polling the operation status."
+              },
+              "x-ms-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "An opaque, globally-unique, server-generated string identifier for the request."
+              }
+            }
+          },
+          "default": {
```

Comment:
Error ("default") responses should also define an `x-ms-request-id` response header.